acquia_cms.profile

```diff
@@ -131,20 +123,34 @@ function acquia_cms_install_tasks(): array {
  */
 function install_acms_send_heartbeat_event() {
   $telemetry = Drupal::classResolver(AcquiaTelemetry::class);
-  $telemetry_service = \Drupal::service('acquia_connector.telemetry');
+  $telemetry_service = \Drupal::service('acquia_cms_common.telemetry');
   $config = \Drupal::config('cohesion.settings');
   $cohesion_configured = $config->get('api_key') && $config->get('organization_key');
-  \Drupal::configFactory()
-    ->getEditable('acquia_connector.settings')
-    ->set('spi.amplitude_api_key', 'e896d8a97a24013cee91e37a35bf7b0b')
-    ->save();
-  $telemetry_service->sendTelemetry('acquia_cms_installed', [
-    'Application UUID' => Environment::getAhApplicationUuid(),
-    'Site Environment' => Environment::getAhEnv(),
-    'Install Time' => $telemetry->calculateTime('install_start_time', 'install_end_time'),
-    'Rebuild Time' => $telemetry->calculateTime('rebuild_start_time', 'rebuild_end_time'),
-    'Site Studio Install Status' => $cohesion_configured ? 1 : 0,
-  ]);
+  $appUuid = Environment::getAhApplicationUuid();
+  $siteGroup = Environment::getAhGroup();
+  $env = Environment::getAhEnv();
+  $acsfStatus = Environment::isAcsfEnv();
+  $siteUri = end(explode('/', \Drupal::getContainer()->getParameter('site.path')));
+  $siteName = \Drupal::config('system.site')->get('name');
+  $version = \Drupal::service('extension.list.module')->getExtensionInfo('acquia_cms')['version'];
+  $telemetryData = [
+    'acquia_cms' => [
+      'application_uuid' => $appUuid,
+      'application_name' => $siteGroup,
+      'environment_name' => $env,
+      'acsf_status' => $acsfStatus,
+      'site_uri' => $siteUri,
+      'site_name' => $siteName,
```

Comment:
If you think above suggestion is good to go then do update this
'site_name' => \Drupal::config('system.site')->get('name'),